source/systems/common.rst

```diff
@@ -64,5 +64,1789 @@ RDHPCS Platforms
+   Submitted batch job 26990
+   jfe01.%
+
+
+   jfe01.% echo scontrol update job 26990 qos=batch | at -M now +5min
+   warning: commands will be executed using /bin/sh
+   job 6 at Sun Dec 17 16:07:00 2023
+   jfe01.%
+
+
+You can change the QOS of all your pending job(s) in a QOS to another QOS after
+it has been pending for a certain time.  The following example script will
+change all your pending “windfall” jobs to “batch” if they have been pending
+for at least 600 seconds (10 min), whenever you run it. <span style="font-size:
+16px">** NOTE:** Do not use a time less than 120 seconds (2 min).
+
+.. code-block::shell
+
+   Script: windfall2batch.sh
+   —-------------
+   #!/bin/bash
+   # Purpose: Look at my jobs in windfall QOS that are still pending.
+   #          If any job has been pending
+   #          for more than "time-in-seconds" move those jobs to "batch"
+   #
+   # Usage: windfall2batch.sh [time-in-seconds]
+   # Default time is 600 seconds (10 minutes)
+
+   time=${1:-600}
+   squeue -u $USER -t pending -q windfall -O JobID,PendingTime,Reason --noheader | \
+      awk -v time=$time '$2 > time && ($3 == "Priority" || $3 == "Resources")      \
+            {system("scontrol update job " $1 " qos=batch")}'
+
+
+You can run the script manually when necessary. If you want to do so routinely,
+you may put this in cron at the desired intervals.
+
+Specifying a job Name
+^^^^^^^^^^^^^^^^^^^^^
+
+Giving your jobs meaningful names can help you locate them when monitoring
+their progress. Use the -J (--job-name) option. For example,
+
+.. code-block:: shell
+
+   #SBATCH -J
+   WRF_ARW_00Z
+
+The default name for a job is the name of the job script
+that is being submitted.
+
+Setting the names of output files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you do not specify the names of the output files that contain the stdout and
+stderr from your job script, a file will be written to the directory in which
+you issued the sbatch command. A file containing both the stdout and stderr
+from your job script will be called: slurm-&lt;jobid&gt;.out where
+&lt;jobid&gt; is the SLURM job id of the job.
+
+Use the -o (--output) option to specify the name of the stdout file <pre>
+#SBATCH -o /full/path/of/stdout/file </pre> Use the -e (--error) option to
+specify the name of the stderr file <pre> #SBATCH -e /full/path/of/stderr/file
+</pre> If you want stdout and stderr to go to the same file, do not specify the
+-e option.
+
+Passing environment variables to the job
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default the environment variables set in the current shell is passed to the
+job that is submitted. However if any variable is explicitly passed into the
+script with a value, only that value is passed to the script!
+
+If you wish to pass local environment to the script and in addition set a
+specific variable that is currently not in the current environment
+(&quot;ndays=20&quot; in the example below), you can do it in the following
```

Comment:
Change `&quot;` to double backticks.

Reply:
Wordy is the least of it. I rewrote the paragraph as per your instructions. It's not the only one that needs an edit.